src/components/buttons/ha-progress-button.ts

```diff
@@ -16,6 +16,12 @@ export class HaProgressButton extends LitElement {
 
   @state() private _result?: "success" | "error";
 
+  constructor() {
+    super();
+
+    this.addEventListener("click", this._buttonTapped);
```

Comment:
isn't this what is causing `_buttonTapped` to be called when disabled? I don't see why we need another listener

Reply:
This is the listener to the host, _buttonTapped was just listening to the mwc-button.
But the @click event was on the host element, so `_buttonTapped` wasn't even called.

Reply:
Set `pointer-events: none;` on `:host`, it is will no longer create click events

Reply:
probably remove the `@click` on the button then. It should bubble to the host, I think

Reply:
Yeah, that click handler should not be needed, the button is disabled already

Reply:
The handler was for checking if it is in progress and then don't forward the click

Reply:
> The handler was for checking if it is in progress and then don't forward the click

But the button is disabled in that case too, so should not fire events

Reply:
Yes